pkg/rewards/claim.go

```diff
@@ -113,29 +117,108 @@ func Claim(cCtx *cli.Context, p utils.Prompter) error {
 		http.DefaultClient,
 	)
 
-	claimDate, rootIndex, err := getClaimDistributionRoot(ctx, config.ClaimTimestamp, elReader, logger)
+	yamlFile, err := os.ReadFile(config.BatchClaimFile)
 	if err != nil {
-		return eigenSdkUtils.WrapError("failed to get claim distribution root", err)
+		return eigenSdkUtils.WrapError("failed to read YAML config file", err)
+	}
+
+	var claimConfigs []struct {
+		EarnerAddress  string   `yaml:"earner_address"`
+		TokenAddresses []string `yaml:"token_addresses"`
+	}
+
+	err = yaml.Unmarshal(yamlFile, &claimConfigs)
+	if err != nil {
+		return eigenSdkUtils.WrapError("failed to parse YAML config", err)
+	}
+
+	var elClaims []rewardscoordinator.IRewardsCoordinatorRewardsMerkleClaim
+
+	for _, claimConfig := range claimConfigs {
+		earnerAddr := gethcommon.HexToAddress(claimConfig.EarnerAddress)
+
+		var tokenAddrs []gethcommon.Address
+		for _, addr := range claimConfig.TokenAddresses {
```

Comment:
we need to also handle the case where they just only provide earner addresses and we need to fetch all the available tokens and then create and submit claims.